src/help/zaphelp/contents/cmdline.html

```diff
@@ -9,6 +9,14 @@
 <BODY BGCOLOR="#ffffff">
 <H1>Command Line</H1>
 
+To run ZAP via the command line, you will need to locate the `zap.sh` script. On Mac, this will be at:<br/>

```

Comment:
The word "need" might be a little misleading, ZAP can be run via the command line without the scripts (i.e. using directly the zap.jar).